definitions/divorce/json/ComplexTypes/ComplexTypes-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "05/11/2020",
+    "ID": "confirmProcessServerServiceDate",
+    "ListElementCode": "ProcessServerServiceDate",
```

Comment:
Why do you need this element?

Reply:
I dont, it was just a misunderstanding of the ACs at the time. Thanks for highlighting

Reply:
Removing it completely